src/js/components/modals/AppModalComponent.jsx

```diff
@@ -386,7 +385,9 @@ var AppModalComponent = React.createClass({
             </StoreFormGroupComponent>
             <div className="row full-bleed">
               <CollapsiblePanelComponent title="Docker container settings">
-                <ContainerSettingsComponent model={model} errors={errors} />
+                <ContainerSettingsComponent
+                  container={state.fields.container}
```

Comment:
We don't need to follow the structure of the application definiton,
we could just passing the fields object here.